.gitignore

```diff
@@ -102,3 +102,4 @@ target/
 
 # Pycharm
 .idea/
+docs/source/_build/
```

Comment:
Please put this line at the top of the file with all the other project-specific excludes